services/worker-manager/test/fixtures/schemas/sample-aws-config.json

```diff
@@ -1,12 +1,4 @@
 {
-  "workerPoolId": "delete/worker",
```

Comment:
I trust it the indentation is ok after the deletion? It's hard to tell from the github ui

#nonblocking #preferential

Reply:
It seems to be ok!